src/c++/perf_analyzer/genai-pa/genai_pa/llm_inputs/llm_inputs.py

```diff
@@ -179,23 +170,16 @@ def _download_dataset(cls, configured_url, starting_index, length) -> Response:
         return dataset
 
     @classmethod
-    def _convert_input_dataset_to_generic_json(
-        cls, input_format: InputFormat, dataset: Response
-    ) -> Dict:
+    def _convert_input_dataset_to_generic_json(cls, dataset: Response) -> Dict:
         dataset_json = dataset.json()
         try:
             LlmInputs._check_for_error_in_json_of_dataset(dataset_json)
         except Exception as e:
             raise GenAiPAException(e)
 
-        if input_format == InputFormat.OPENAI:
-            generic_dataset_json = LlmInputs._convert_openai_to_generic_input_json(
-                dataset_json
-            )
-        else:
-            raise GenAiPAException(
-                f"Input format {input_format} is not supported at this time"
-            )
+        generic_dataset_json = LlmInputs._convert_openai_to_generic_input_json(
```

Comment:
I think this method changes to _convert_dataset_json_generic_input_json

Reply:
Fixed

Reply:
Don't have any output formats that are not supported